ci/jenkins/test-mc.sh

```diff
@@ -257,8 +258,8 @@ function run_multicluster_e2e {
 
     wait_for_multicluster_controller_ready
 
-    docker pull "${DOCKER_REGISTRY}/antrea/nginx"
-    docker tag "${DOCKER_REGISTRY}/antrea/nginx:latest" "nginx:latest"
+    docker pull $NGINX_IMAGE
+    docker tag $NGINX_IMAGE "nginx:latest"
```

Comment:
Although it's not a big deal, could we use the original tag to indicate this is different from the real nginx:latest image?
I mean replace "nginx:latest" with "nginx:1.14-2" here and in multicluster/test/e2e/framework.go